Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.Events.cs

```diff
@@ -33,6 +35,10 @@ public partial class Menu
         private bool _altHandled;
         private bool _isLostFocus = true;
         private Control _lastFocusElementBeforeMenu;
+        private double _x1;
```

Comment:
Using actual width\height with force using + inside the if condition and this condition is repeated many times so having it pre-calculated would be better.
using Points is a good suggestion.